### PR TITLE
HPCC-14993 Return Protect info in WsDfu.DFUQuery and DFUInfo

### PR DESCRIPTION
**Pull request by wangkx**
Signed-off-by: wangkx kevin.wang@lexisnexis.com

### CODE REVIEW

**Comment by wangkx** [comment]
@afishbeck please review.

File: esp/services/ws_dfu/ws_dfuService.cpp [hunk 1]
```diff
@@ -1851,6 +1851,26 @@ void CWsDfuEx::doGetFileDetails(IEspContext &context, IUserDescriptor* udesc, co
             FileDetails.setNodeGroup(cluster);
         else if (clusters.length() == 1)
             FileDetails.setNodeGroup(clusters.item(0));
+        if (version >= 1.34)
```
Comment (afishbeck) [inline esp/services/ws_dfu/ws_dfuService.cpp]: This isn't that much information or overhead.  So you really don't need this check. But it's ok if you prefer to leave it for this.

File: esp/services/ws_dfu/ws_dfuService.cpp [hunk 2]
```diff
@@ -3320,6 +3340,11 @@ bool CWsDfuEx::addToLogicalFileList(IPropertyTree& file, const char* nodeGroup,
                 lFile->setPersistent(true);
             if (file.hasProp(getDFUQResultFieldName(DFUQRFsuperowners)))
                 lFile->setSuperOwners(file.queryProp(getDFUQResultFieldName(DFUQRFsuperowners)));
+            if (version >= 1.34)
```
Comment (afishbeck) [inline esp/services/ws_dfu/ws_dfuService.cpp]: This check isn't necessary.

**Comment by wangkx** [comment]
@afishbeck the version checks are removed. Please review.

**Comment by afishbeck** [comment]
@wangkx looks good

**Comment by wangkx** [comment]
@richardkchapman this PR is ready to be merged.

**Pull request merged**